source/_integrations/light.mqtt.markdown

```diff
@@ -69,11 +69,19 @@ availability:
       description: An MQTT topic subscribed to receive availability (online/offline) updates.
       required: true
       type: string
+    value_template:
```

Comment:
Please make sure json and template schemas are updated too.

Reply:
Updated the JSON schema, cannot find any template schema here.

Reply:
found it, fixed